src/utils/inlineImage.ts

```diff
@@ -11,12 +11,14 @@ import { getSignedUrlUpload, getSignedUrlFile } from '@/app/actions'
 
 export const uploadImageHandler = async (file: File, token: string, task_id: string | null): Promise<string> => {
   const supabaseActions = new SupabaseActions()
-  const fileName = generateRandomString(file.name)
 
+  const fileName = generateRandomString(file.name)
   const signedUrl: ISignedUrlUpload = await getSignedUrlUpload(token, fileName)
-  const filePayload = await supabaseActions.uploadAttachment(file, signedUrl, task_id)
+  const { filePayload, error } = await supabaseActions.uploadAttachment(file, signedUrl, task_id)
   const url = await getSignedUrlFile(token ?? '', filePayload?.filePath ?? '')
-
+  if (error) {
```

Comment:
if token or filePayload.filePath is empty we don't want to get a signed url file from it in the first place. So handling it separately before this step would be the best to prevent unintended side effects